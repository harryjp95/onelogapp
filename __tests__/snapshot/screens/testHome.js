import 'react-native'
import React from 'react'
import renderer from 'react-test-renderer'

import HomeScreen from '../../../src/Components/Screens/Home'

it('renders correctly', () => {
  const homeScreen = renderer.create(<HomeScreen />)

  expect(homeScreen).toMatchSnapshot()
})
