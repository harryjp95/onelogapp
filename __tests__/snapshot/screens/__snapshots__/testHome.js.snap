// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#ebebeb",
      "flexDirection": "column",
      "height": 1214,
      "justifyContent": "flex-start",
      "width": 750,
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    />
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 5,
          "flexDirection": "column",
          "justifyContent": "center",
        }
      }
    >
      <ActivityIndicator
        animating={true}
        color="black"
        hidesWhenStopped={true}
        size="small"
      />
    </View>
    <BVLinearGradient
      colors={
        Array [
          4293651435,
          4286611584,
          4293651435,
        ]
      }
      endPoint={
        Object {
          "x": 1,
          "y": 1,
        }
      }
      locations={null}
      startPoint={
        Object {
          "x": 0,
          "y": 1,
        }
      }
      style={
        Object {
          "alignItems": "center",
          "height": 0.5,
          "justifyContent": "center",
          "width": "100%",
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    />
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 5,
          "flexDirection": "column",
          "justifyContent": "center",
        }
      }
    >
      <ActivityIndicator
        animating={true}
        color="black"
        hidesWhenStopped={true}
        size="small"
      />
    </View>
    <BVLinearGradient
      colors={
        Array [
          4293651435,
          4286611584,
          4293651435,
        ]
      }
      endPoint={
        Object {
          "x": 1,
          "y": 1,
        }
      }
      locations={null}
      startPoint={
        Object {
          "x": 0,
          "y": 1,
        }
      }
      style={
        Object {
          "alignItems": "center",
          "height": 0.5,
          "justifyContent": "center",
          "width": "100%",
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    />
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 5,
          "flexDirection": "column",
          "justifyContent": "center",
        }
      }
    >
      <ActivityIndicator
        animating={true}
        color="black"
        hidesWhenStopped={true}
        size="small"
      />
    </View>
    <BVLinearGradient
      colors={
        Array [
          4293651435,
          4286611584,
          4293651435,
        ]
      }
      endPoint={
        Object {
          "x": 1,
          "y": 1,
        }
      }
      locations={null}
      startPoint={
        Object {
          "x": 0,
          "y": 1,
        }
      }
      style={
        Object {
          "alignItems": "center",
          "height": 0.5,
          "justifyContent": "center",
          "width": "100%",
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    />
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 5,
          "flexDirection": "column",
          "justifyContent": "center",
        }
      }
    >
      <ActivityIndicator
        animating={true}
        color="black"
        hidesWhenStopped={true}
        size="small"
      />
    </View>
    <BVLinearGradient
      colors={
        Array [
          4293651435,
          4286611584,
          4293651435,
        ]
      }
      endPoint={
        Object {
          "x": 1,
          "y": 1,
        }
      }
      locations={null}
      startPoint={
        Object {
          "x": 0,
          "y": 1,
        }
      }
      style={
        Object {
          "alignItems": "center",
          "height": 0.5,
          "justifyContent": "center",
          "width": "100%",
        }
      }
    />
  </View>
</View>
`;
